uktrade/data-hub-frontend
# Add filter to ReferralList

## Description of change

Adds a dropdown to _My referrals_ by which you can choose whether all, sent or received referrals will be listed.

## Test instructions

1. Go to the homepage
2. Select the _My referrals_ tab
3. The _All referrals_ option of the _View_ labeled dropdown should be selected by default and all referrals should be displayed in the list
4. Select one fo the other options of the dropdown and verify that the list reflects the selected filter
5. Repeat steps 3 and 4 for the remaining dropdown option

## Screenshots

### Before
<img width="587" alt="Screen Shot 2020-04-09 at 2 49 57 PM" src="https://user-images.githubusercontent.com/2333157/78902757-1f682200-7a72-11ea-81f4-940b9f606b98.png">

### After
<img width="587" alt="Screen Shot 2020-04-09 at 2 53 38 PM" src="https://user-images.githubusercontent.com/2333157/78902758-2000b880-7a72-11ea-804a-a27b33cbd167.png">
